deepspeed/module_inject/containers/base.py

```diff
@@ -347,6 +347,10 @@ def copy_data_to_new_module(self):
         self.module.norm_w.data.copy_(self.input_nw.to(get_accelerator().current_device_name()))
         self.module.norm_b.data.copy_(self.input_nb.to(get_accelerator().current_device_name()))
 
+    def update_merged_qkv(self):
+        if hasattr(self, '_update_merged_qkv'):
+            self.update_merged_qkv()
```

Comment:
`self.update_merged_qkv()` should be `self._update_merged_qkv()` so that the Megatron feature function is called.